hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/HoodieHiveUtils.java

```diff
@@ -40,6 +44,8 @@
   public static final String HOODIE_CONSUME_MODE_PATTERN = "hoodie.%s.consume.mode";
   public static final String HOODIE_START_COMMIT_PATTERN = "hoodie.%s.consume.start.timestamp";
   public static final String HOODIE_MAX_COMMIT_PATTERN = "hoodie.%s.consume.max.commits";
+  public static final String HOODIE_CONSUME_PENDING_COMMITS = "hoodie.%s.consume.pending.commits";
+  public static final String HOODIE_CONSUME_VALIDATE_TIMESTAMP = "hoodie.%s.consume.validate.commit";
```

Comment:
s/hoodie.%s.consume.validate.commit/hoodie.%s.consume.commit

We can just treat this config for time-travel. if consume.pending.commits is enabled, this time-travel will work on inflight, if it's disabled, this can time-travel to a completed commit. For now, you can just implement this for your use-case and we don't need to leak the specific use-case like "validation" into the naming.